roles/confluent.common/tasks/main.yml

```
@@ -31,6 +31,24 @@
     state: absent
     regexp: '^Defaults\s+requiretty.*$'
 
+# Process Archive File deployments
+- name: Create Confluent Platform install directory
+  file:
+    path: "{{archive_destination_path}}"
+    state: directory
+    mode: 0755
+  when: installation_method == "archive"
+
+# If the target directory (i.e. creates) doesn't exist then download and expand the remote archive into target
+- name: Expand remote Confluent Platform archive
+  unarchive:
+    src: "{{ confluent_archive_file_path }}"
+    remote_src: yes
```

```suggestion
    remote_src: true
```

yaml linter complains

---

Fixed in d10c97e8